docs/overview.yaml

```
@@ -420,5 +420,5 @@ Logic and computation:
     first-order formula: 'first_order.language.formula'
     satisfiability: 'first_order.language.Theory.is_satisfiable'
     substructure: 'first_order.language.substructure'
-    definable set: 'first_order.language.definable_set'
+    definable set: 'set.definable_set'
```

```suggestion
    definable set: 'set.definable'
```
Maybe it's better to refer to the unbundled notion, since that will almost certainly have more API?